applications/sofa/gui/BaseViewer.cpp

```diff
@@ -132,6 +133,9 @@ void BaseViewer::setPrefix(const std::string& prefix, bool prependDirectory)
 #ifdef SOFA_HAVE_FFMPEG
     videoRecorder.setPrefix(fullPrefix);
 #endif
+#ifdef SOFA_HAVE_FFMPEG_EXEC
```

Comment:
I'm not very found of #ifdef #endif  is wonder if there is no cleaner alternative without this kind of code branches.